perseo-main/src/main/java/com/telefonica/iot/perseo/LogLevelServlet.java

```diff
@@ -95,11 +103,18 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
             throws ServletException, IOException {
         logger.debug("getting log level");
         synchronized (mutex) {
-            String currentLevel = LogManager.getRootLogger().getLevel().toString();
-            response.setCharacterEncoding("UTF-8");
-            response.setContentType("application/json");
-            response.setStatus(HttpServletResponse.SC_OK);
-            response.getOutputStream().print("{\"level\":\""+ currentLevel +"\"}");
+             try {
```

Comment:
Wrong indent

Reply:
Fixed in 47ab83625cbfba55660da30a785723e2fd7ecf55

Reply:
Travis ci fixed in b4a4d739cc56604002ce38463fa15aeccb64b823

Reply:
Note really fixed... indent is not correct. An extra whitespace before try should be removed:

![imagen](https://user-images.githubusercontent.com/1534240/73432146-6800b200-4342-11ea-8ee5-698c7af85443.png)